tests/cpp/test_alias.cpp

```diff
@@ -1395,4 +1395,36 @@ TEST_F(AliasTest, SegmentMetaOps) {
   }
 }
 
+TEST_F(AliasTest, StackedAliases) {
+  auto fusion = std::make_unique<Fusion>();
+  FusionGuard fg(fusion.get());
+
+  TensorView* tv0 = makeContigConcreteTensor({2, 3});
+  TensorView* tv1 = slice(tv0, {0, 0}, {2, 3});
+  TensorView* tv2 = segment_set(tv1);
+  // segment_set adds a segment
+  TensorView* tv3 = slice(tv2, {0, 0}, {2, 3});
+  TensorView* tv4 = segment_set(tv3);
+  // segment_set adds a segment
+  TensorView* tv5 = relu(tv4);
+  TensorView* tv6 = slice(tv5, {0, 0}, {2, 3});
+  TensorView* tv7 = broadcast(tv6, {false, true, false});
```

Comment:
Without the code change, we would not be able to mark `tv7` as an aliased from tv5.